### ISSUE TITLE
Make distinction custom and not non custom server more visible in onboarding

### ISSUE THREAD

**Issue by ann0see** [gkasdorf/memmy#221]
**Is your feature request related to a problem? Please describe.**
Currently the Hubs page in the onboarding shows the Join and Login button right at the top. Right below there's the list of servers. It could be confusing for people tapping on "Join" and then being presented with a server url they don't know - as they don't have one yet. I believe it's an advanced feature and shouldn't be that prominent. 

**Describe the solution you'd like**

* Rename the Join button to Join custom server or Join custom hub and move somewhere else (e.g below all default servers) - see it as advanced feature. However the Login button should still be visible and not moved
* have a clear separation with a line between the servers and the custom login
* Move the custom login and Join somewhere down (not that preferred)

**Additional context**

![image](https://github.com/gkasdorf/memmy/assets/20726856/f1e84a8f-c93b-4ace-9462-79dae48ce5fc)

**Comment by gkasdorf** [comment]
Good idea. I'll put an additional option down below the list for joining a custom server.
